resources/views/inputs/password-toggle.blade.php

```diff
@@ -7,31 +7,36 @@
         @endif
 
         <div class="input-wrapper">
-            <div class="flex space-x-4">
-                <div class="flex-1">
-                    <input
-                        x-ref="password-input"
-                        x-bind:type="show ? 'text' : 'password'"
-                        id="{{ $id ?? $name }}"
-                        name="{{ $name }}"
-                        class="input-text @error($name) input-text--error @enderror {{ $inputClass ?? '' }}"
-                        wire:model="{{ $model ?? $name }}"
-                        @if($max ?? false) maxlength="{{ $max }}" @endif
-                        @if($value ?? false) value="{{ $value }}" @endif
-                        @if($autofocus ?? false) autofocus @endif
-                        @if($readonly ?? false) readonly @endif
-                        @if($required ?? false) required @endif
-                    />
-                </div>
-                <div>
-                    <button type="button" class="w-12 h-12 button-icon" @click="toggle()">
-                        <span x-show="!show">@svg('view', 'w-6 h-6')</span>
-                        <span x-show="show" x-cloak>@svg('hide', 'w-6 h-6')</span>
-                    </button>
-                </div>
-            </div>
-        </div>
+            {{--input--}}
+            <input
+                x-ref="password-input"
+                :type="type"
+                id="{{ $id ?? $name }}"
+                name="{{ $name }}"
+                class="input-text shifted @error($name) input-text--error @enderror {{ $inputClass ?? '' }}"
+                wire:model="{{ $model ?? $name }}"
+                @if($max ?? false) maxlength="{{ $max }}" @endif
+                @if($value ?? false) value="{{ $value }}" @endif
+                @if($autofocus ?? false) autofocus @endif
+                @if($readonly ?? false) readonly @endif
+                @if($required ?? false) required @endif
+                @keydown="$dispatch('typing')"
+            />
+
+            {{--toggle--}}
+            <button type="button" class="right-0 px-4 input-icon text-theme-primary-300 @error($name) text-theme-danger-500 @enderror" @click="toggle()">
```

Comment:
@ItsANameToo on all 4 corners? or only the right side?

Reply:
all 4, not sure if it will look good as I now see the button lines extends slightly past the input. Can you try and post a screenshot of how it ends up looking with rounded corners?

Reply:
@ItsANameToo 
![image](https://user-images.githubusercontent.com/2118799/125431289-917a0d41-0f8b-4a00-b24f-632e024df51c.png)
![image](https://user-images.githubusercontent.com/2118799/125431313-f434f750-9e68-47d6-817a-3a3fd5dddf22.png)

Reply:
much better

Reply:
yeah, ship it :oleglala: